### DISCUSSION TITLE
工研院研發 智慧監測系統💻

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS4719-C/Python-Community-GS4719-C#735] [General]
新聞來源：
https://finance.technews.tw/2022/10/04/plastic-product/

新聞概要：
根據統計，台灣事業單位每年約產出 24 萬公噸塑膠廢棄物，僅約 28% 回收，其餘則焚化處理，造成環境負擔，因此工研院研發「射出成型智慧監測系統」，協助加工產線減少 30% 的不良塑膠廢品，藉由源頭減廢，邁向淨零排放。

心得：
經由導入中南部的實例過後，若是之後台灣的多數工業廠商都能應用這個智慧監測系統，這樣就能大幅下降垃圾處理量，而且也能為企業增加產品良率！

![image](https://user-images.githubusercontent.com/90883526/193984034-5c0f6eff-9e50-4590-9960-b2863504fd8f.jpeg)

**Comment by moonnight3503** [comment]
環保愛地球~

**Comment by danwan1027** [comment]
人類每天製造那麼多污染，能有這樣的發明這的太棒了

**Comment by dinosaur1009** [comment]
好奇系統內部怎麼運作的！如何去執行！這真的是個很酷的發明而且很實用

**Comment by Yunhans** [comment]
塑膠真的是一個方便但是對環境不好的產品

**Comment by Chiway1229** [comment]
台灣的塑膠製品確實對環境造成很大的負擔了
有了這項設施 搭配上台灣人分類的好習慣
相信能減緩污染的速度

**Comment by williamyen0623** [comment]
環保讚讚！

**Comment by a45320614** [comment]
真是偉大的發明，期望這個系統能在台灣各地普及，提升環保效率

**Comment by Pietra1226** [comment]
很棒很實用的發明，為地球環境盡一份心力

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
隨者科技的進步 但也要好好的看看我們身邊的環境 這樣才能走的長久

**Comment by delia07152** [comment]
雖然這項技術我覺得很好，但我認為處理根本的問題才是解決之道，不過現在各國都有發展減塑的政策，希望有朝一日塑膠完全得被某個更好的產品取代!

**Comment by BlahFish1006** [comment]
從源頭的工廠製造減少塑膠製品不必要的浪費及垃圾汙染，是一個很有意義的應用，也能有效減少工廠的不良品成本。
希望台灣能在替代材質的方案待被研發出來的過渡期，將這個技術應用普及化。

**Comment by manipulateed** [comment]
相信不只塑膠，廢水或是空汙等等環境危害都能應用到此技術，讓世界邁向更友善的未來

**Comment by zhiron544** [comment]
這個系統真的很厲害，能夠在源頭就減少塑膠製品的浪費和污染，也能節省工廠的成本和資源。我覺得這個系統不只適用於塑膠製品，也可以應用在其他的材料或是產業上，讓台灣的製造業更加智慧和綠色。
